src/aktualizr_secondary/aktualizr_secondary.cc

```diff
@@ -86,22 +87,23 @@ bool AktualizrSecondary::putMetadataResp(const Uptane::RawMetaPack& meta_pack) {
       target_ = std_::make_unique<Uptane::Target>(*it);
     }
   }
-  storage_->storeMetadata(meta_pack);
+  storage_->storeRole(meta_pack.director_root, Uptane::RepositoryType::Director, Uptane::Role::Root(),
```

Comment:
Right now implemenation is the same for both repos in both `FSStorage` and `SQLStorage`. What might be worth splitting is separating `storeRoot`/`loadRoot` from `storeNonRootRole`/`loadNonRootRole`, because we actually need versions only for root.